packages/lab/src/list-next/ListNext.css

```diff
@@ -0,0 +1,16 @@
+/* Style applied to the root element */
+.saltListNext {
+  margin: 0;
+  background: var(--salt-container-primary-background);
+  border-color: var(--salt-container-primary-borderColor);
+  border-style: var(--salt-container-borderStyle);
+  border-width: var(--salt-size-border);
+  height: 100%;
+  outline: none;
+  overflow-y: auto;
+  overflow-x: hidden;
```

Comment:
Do we need to prevent x overflow here?

Reply:
I think so, if we have overflow auto in one side (and really, it is the intent for that to happen) it shouldn't scroll on the other side